app/controllers/content_change_requests_controller.rb

```diff
@@ -16,11 +16,14 @@ def parse_request_from_params
   def content_change_request_params
     params.require(:support_requests_content_change_request).permit(
       :title,
+      :reason_for_change,
+      :subject_area,
       :details_of_change,
+      :why_is_change_needed,
       :url,
       :related_urls,
       requester_attributes: %i[email name collaborator_emails],
-      time_constraint_attributes: %i[not_before_date needed_by_date time_constraint_reason],
+      time_constraint_attributes: %i[not_before_date needed_by_date time_constraint_reason needed_by_time not_before_time],
```

Comment:
Hey 👋 apparently these two new fields aren't making it through to Zendesk. I think we'll need to add them to [zendesk_ticket.rb](https://github.com/alphagov/support/blob/98f88ef5dba31485aae195939573ec8114d68866/app/models/zendesk/zendesk_ticket.rb#L48-L58).